README.md

```diff
@@ -237,7 +250,7 @@ The payload should look something like:
 ### docker-compose
 
 Multiple docker-compose files are provided:
-- a minimal `docker-compose.yml` with PostgreSQL
+- a minimal `docker-compose.yml` with two PostgreSQL containers (one for catalog and metadata, the other for converted CSV to database)
```

Comment:
Not sure, since it works with default configuration:

```
DATABASE_URL = "postgres://postgres:postgres@localhost:5432/postgres"
DATABASE_URL_CSV = "postgres://postgres:postgres@localhost:5434/postgres"
```

I don't see the value in adding yet another file.